janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/JanusGraphConstants.java

```diff
@@ -73,6 +78,11 @@
             if (!v.isEmpty()) b.add(v);
         }
         COMPATIBLE_VERSIONS = b.build();
+        
+        for (String v : props.getProperty("titan.compatible-versions", "").split(",")) {
```

Comment:
I got that sure. I just followed previous implementation, shall I change from line 74 til 86 with this? Titan had this before so I just followed it to keep it consistent.